bioblend/galaxy/jobs/__init__.py

```diff
@@ -48,7 +61,16 @@ def get_jobs(self, state=None, tool_id=None, user_details=False,
           filter on.
 
         :type history_id: str
-        :param history_id: Encoded history ID to filter on.
+        :param history_id: Limit listing of jobs to those that are associated
+          with the history_id. If None, jobs are returned for all histories.
+
+        :type workflow_id: string
+        :param workflow_id: Limit listing of jobs to those that are associated
+          with the workflow_id. If None, jobs are returned for all workflows.
+
+        :type invocation_id: string
+        :param invocation_id: Limit listing of jobs to those that are associated
+          with the invocation_id. If None, jobs are returned for all invocations.
 
```

Comment:
Could you add a note here, similar to https://github.com/galaxyproject/bioblend/blob/main/bioblend/galaxy/jobs/__init__.py#L160, explaining which filtering options are available in which Galaxy version?